data/json/items/armor/helmets.json

```
@@ -255,6 +255,37 @@
     ],
     "melee_damage": { "bash": 2 }
   },
+  {
+    "id": "helmet_army_outdated",
+    "repairs_like": "helmet_barbute",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "flak helmet" },
+    "description": "An M1 combat helmet, famously worn by American soldiers in the Second World War as well as the Korean and Vietnam wars.  Its steel construction has rendered it outdated by modern military standards, but it still remains popular among geurilla fighters, collectors and war re-enactors.",
+    "weight": "1400 g",
+    "volume": "1900 ml",
+    "longest_side": "28 cm",
+    "price": 20000,
+    "//": "assuming this is an antique and not a cheap replica",
+    "price_postapoc": 1200,
+    "to_hit": -1,
+    "material": [ "steel", "cotton" ],
```

```suggestion
    "material": [ "hc_steel", "cotton" ],
    "repairs_with": [ "hc_steel" ],
```
To prevent cheesing repairs by only repairing the non-protective part.